### PR TITLE
React capstone Project

### PR DESCRIPTION
**Pull request by Bellagirl-maker**
- Create an App that renders API data about Covid-19 worldwide
- Create unit and integrated tests 
- Include a video recording link in README.md file 
- Include a live link of the site deployed on netlify in README. md file

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
```
Comment (ShadyShawkat) [inline src/index.css]: - [ ] Kindly make sure to follow the same typography used in the design. As per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.) requirement.

File: src/tests/countriesSlice.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      { country: 'Kenya', cases: 5400 },
+    ];
+
+    jest.spyOn(axios, 'get').mockResolvedValueOnce({ data: responseData });
+
+    const action = fetchCountries('countries');
+    const result = await action.payload;
+    expect(result).toBeFalsy();
+  });
+});
```
Comment (ShadyShawkat) [inline src/tests/countriesSlice.test.js]: - [ ] Please ensure that the tests are structured correctly. The name of this test implies that you're testing for a successful response, however, you're testing if the payload is "falsy" which contradicts the former statement.

File: README.md [hunk 3]
```diff
@@ -1 +1,171 @@
...
+I would like to thank all my colleagues and code reviewers in microverse for helping me.
+This capstone project is based on the original design idea of <a href="https://www.behance.net/sakwadesignstudio>Nelson Sakwa on Behance.</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (ShadyShawkat) [inline README.md:169]: - [ ] Please make sure that the MIT file is there and working properly when this link is clicked.

File: src/tests/Details.test.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```
Comment (ShadyShawkat) [inline src/tests/Details.test.js:1]: - [ ] Please add more integration tests that test for user interactions across the app, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests%20for%20your%20application%20using%20the%20React%20Testing%20Library.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Bellagirl-maker, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (Bellagirl-maker) [reply src/index.css]: @ShadyShawkat done

File: src/tests/countriesSlice.test.js (hunk 2 above)
Comment (Bellagirl-maker) [reply src/tests/countriesSlice.test.js]: @ShadyShawkat  done

File: README.md (hunk 3 above)
Comment (Bellagirl-maker) [reply README.md:169]: @ShadyShawkat done

File: src/tests/Details.test.js (hunk 4 above)
Comment (Bellagirl-maker) [reply src/tests/Details.test.js:1]: @ShadyShawkat I have tried all I possibly can but the tests are not passing so I have removed them

**Review by TadesseDev — APPROVED** [review]
### Hi @Bellagirl-maker 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**